### DISCUSSION TITLE
Open source contributions [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5675] [Discord Threads]
> 🙋‍♂️ **SuperMan9**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You can pick non-community issues as well. Just get in touch with us before you pick to ensure efforts are not duplicated  
#  
> 🙋‍♂️ **SuperMan9**  
#### sounds good!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1095608140453654658)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
